rapid/rapid.py

```diff
@@ -98,8 +101,43 @@ def wait_for_job_outcome(self, _id: str, interval: int = 1):
                 raise JobFailedException("Upload failed", progress)
             time.sleep(interval)
 
+    def download_dataframe(
+        self, domain: str, dataset: str, version: int = 1, query: Query = Query()
```

Comment:
I don't think version should default to one, instead we should just leave it blank and let the API grab the latest version of the data